### ISSUE TITLE
Data azurerm_servicebus_queue fails to be read with "cannot parse an empty string"

### ISSUE THREAD

**Issue by Cheroukee** [hashicorp/terraform-provider-azurerm#17748]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Community Note

* Please vote on this issue by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

1.2.5

### AzureRM Provider Version

3.15.0

### Affected Resource(s)/Data Source(s)

azurerm_servicebus_queue

### Terraform Configuration Files

```hcl
terraform {
  required_providers {
    azurerm = {
      source  = "hashicorp/azurerm"
      version = "~> 3.15.0"
    }
  }
}

provider "azurerm" {
  features {}
}

data "azurerm_servicebus_queue" "status_messages" {
  name                = "<name of queue>"
  resource_group_name = "<resource group name>"   
  namespace_name      = "<namespace of service bus>"      
}
```

### Debug Output/Panic Output

```shell
data.azurerm_servicebus_queue.status_messages: Reading...
2022-07-25T09:31:59.231-0400 [ERROR] provider.terraform-provider-azurerm_v3.15.0_x5.exe: Response contains error diagnostic: tf_rpc=ReadDataSource @module=sdk.proto diagnostic_summary="parsing "": cannot parse an 
empty string" tf_provider_addr=provider diagnostic_detail= diagnostic_severity=ERROR tf_data_source_type=azurerm_servicebus_queue tf_proto_version=5.2 @caller=github.com/hashicorp/terraform-plugin-go@v0.10.0/tfprotov5/internal/diag/diagnostics.go:56 tf_req_id=3b44889c-d61c-00cd-5641-d946cd20c9b0 timestamp=2022-07-25T09:31:59.231-0400
2022-07-25T09:31:59.231-0400 [ERROR] vertex "data.azurerm_servicebus_queue.status_messages" error: parsing "": cannot parse an empty string
2022-07-25T09:31:59.233-0400 [ERROR] vertex "data.azurerm_servicebus_queue.status_messages" error: parsing "": cannot parse an empty string
2022-07-25T09:31:59.233-0400 [ERROR] vertex "data.azurerm_servicebus_queue.status_messages (expand)" error: parsing "": cannot parse an empty string
2022-07-25T09:31:59.233-0400 [INFO]  backend/local: plan operation completed
╷
│ Error: parsing "": cannot parse an empty string
│
│   with data.azurerm_servicebus_queue.status_messages,
│   on dependencies.tf line 2, in data "azurerm_servicebus_queue" "status_messages":
│    2: data "azurerm_servicebus_queue" "status_messages" {
│
╵
```

### Expected Behaviour

The existing resource is read correctly

### Actual Behaviour

The plan fails to complete with the message : "parsing "": cannot parse an empty string" 

### Steps to Reproduce

1. `terraform plan` on the snippet
2. Error will show up

Issue will show up whether the resource exits or not

**Comment by mbfrahry** [comment]
Hi @Cheroukee, sorry for the regression here. We got a fix in with #17755 and just pushed a release out to 3.15.1. Let me know if that fixes up this issue for you!

**Comment by Cheroukee** [comment]
Yes it did ! We can close the issue. Thanks !
